addons/website/static/src/js/text_processing.js

```diff
@@ -554,3 +554,13 @@ export function getOffsetNode(textEl, offset) {
     }
     return [offsetNode, offset - index];
 }
+
+/**
+ * It is mainly used to check if an element is part of a text option structure.
```

Comment:
"It is mainly used to" -> should not be the main explanation of a function. Just say "Checks if ...", seems good.